lib/src/cli/deployapps/baas_client.dart

```diff
@@ -67,7 +67,7 @@ class BaasClient {
         _headers = <String, String>{'Accept': 'application/json'},
         _appSuffix = '-$_clusterName';
 
-  /// A client that imports apps in a MongoDB Realm docker image. See https://github.com/realm/ci/tree/master/realm/docker/mongodb-realm
+  /// A client that imports apps in a MongoDB Atlas docker image. See https://github.com/realm/ci/tree/master/realm/docker/mongodb-realm
```

Comment:
```suggestion
  /// A client that imports apps in an Atlas App Services docker image. See https://github.com/realm/ci/tree/master/realm/docker/mongodb-realm
```

Similarly here - we don't have any Atlas docker images and the AAS docker image is running a local MDB server, not Atlas.